starter/js/atm.js

```diff
@@ -1,3 +1,36 @@
+$(document).ready(function(){
+$('input #deposit').on("click", depositMoney)
+$('input #withdrawal').on("click", depositMoney)
+}
+var deposit = 0;
+var withdrawal = 0;
+var money = $('input').val();
+var balance = 0;
+
+function depositMoney(event){
+  event.preventDefault()
+  if(money > 0){
+      balance = balance + money;
+  }
+  console.log(balance);
+}
+
+function withdrawMoney(event){
+  event.preventDefault()
+  if(money<balance){
+    balance = balance - money;
+  }
+  console.log(balance)
+}
```

Comment:
These functions have been created to withdraw and deposit money but you never specify which account (savings or checkings). Your logic is on par, but looks like you didn't get this working fully quite yet.